### DISCUSSION TITLE
Creating Different ArchWSL Instances by Renaming arch.exe

### DISCUSSION THREAD

**Discussion by like955** [yuk7/ArchWSL#374] [Q&A]
I've discovered that you can create different ArchWSL instances by copying and renaming `arch.exe`. However, I'm concerned about potential issues when updating via Scoop. Specifically, will updating cause problems since the `current` directory will point to the new version? Does Scoop update the `arch.exe` file, and if so, how does this affect the renamed instances?

**Comment by yuk7** [comment]
Yes, Arch.exe is updated by Scoop.
I don't recommend manually modifying files managed by Scoop.
However, if you rename the file after copying and place it outside the Scoop directory, it will not be managed by Scoop and there will be no problem.

**Reply by like955** [reply]
> Yes, Arch.exe is updated by Scoop. I don't recommend manually modifying files managed by Scoop. However, if you rename the file after copying and place it outside the Scoop directory, it will not be managed by Scoop and there will be no problem.

But in this case, new instances won't be updated by Scoop, which means they might fall behind in version (though I'm not sure if this will have serious consequences).

**Comment by rehan5555ak2-rgb** [comment]
[Uploading DataPrivacyFrameworkParticipantsList.xlsx…]()
__

**Comment by rehan5555ak2-rgb** [comment]
###
